terraform.gpaas-azure-migration/container-apps-hosting.tf

```diff
@@ -0,0 +1,41 @@
+module "azure_container_apps_hosting" {
+  source = "github.com/DFE-Digital/terraform-azurerm-container-apps-hosting?ref=v0.9.0"
+
+  environment    = local.environment
+  project_name   = local.project_name
+  azure_location = local.azure_location
+  tags           = local.tags
+
+  virtual_network_address_space = local.virtual_network_address_space
+
+  enable_container_registry = local.enable_container_registry
+
+  image_name                             = local.image_name
+  container_command                      = local.container_command
+  container_secret_environment_variables = local.container_secret_environment_variables
+
+  enable_mssql_database = local.enable_mssql_database
```

Comment:
The app doesn't connect directly to any DB's so perhaps we should remove this one.

Reply:
We'd like to leave in the _potential_ options for enabling these. That way, the terraform code remains predictable across the multiple different DfE services that rely on it. 

You can simply provide this as a `(bool) false` in the `tfvars` file to prevent it being deployed.

e.g.
```
enable_container_registry = true
enable_cdn_frontdoor      = true
enable_mssql_database     = false
enable_redis_cache        = false
```

Reply:
Makes sense. Thanks.